### ISSUE TITLE
More design questions

### ISSUE THREAD

**Issue by mosespray** [feilipu/yaz180#3]
I am curious as to why you set BUSRQ to VCC and leave BUSAK unconnected.  Is it because you are the only only using the bus (i.e. there aren't io cards to compete with)?

On page one of the schema, why do you connect ADDR10 - ADDR19 to A14 - A5 and ADDR5 - ADDR9 to A19 - A15?

**Comment by feilipu** [comment]
> I am curious as to why you set BUSRQ to VCC and leave BUSAK unconnected. Is it because you are the only only using the bus (i.e. there aren't io cards to compete with)?

I made the decision early on in my design to make the YAZ180 a "Single Board Computer". This was because I didn't fully understand the transmission line theory which can result in issues when you try to run high speed signals through connectors and over longer distances with taps and open stubs.

Because the YAZ180 is a SBC, I was therefore aware that nothing on my board would need to control the bus, and therefore the `/BUSREQ` and `/BUSACK` signals were unnecessary. The `/BUSREQ` signal needs to be tied high as it is an active low signal. Active low is the normal case with most of the Z80 (Z180) control signals.

Later, I added the 82C55 PIO device to allow interfacing and expansion off the board. The three 8 bit ports of the 82C55 device are organised to look like an IDE interface, but really this is just for convenience. An IDE cable and connectors makes a very neat and cheap physical interface for system expansion.

> On page one of the schema, why do you connect ADDR10 - ADDR19 to A14 - A5 and ADDR5 - ADDR9 to A19 - A15?

The SRAM device is very compact, and is a SMD. I had quite a bit of difficulty organising the signal flow for the address and data lines to appear in the correct order at the SRAM device, because the pin-out of the Z8S180 PLCC seems to be strange.

After quite a bit of hand-wringing and sleepless nights, I realised that RAM is an acronym for "Random Access Memory", and so it really doesn't make any difference in which order the address lines are connected, provided that for each address a unique byte of data is stored or retrieved.

This comment in my hardware design notes...

"Getting 20 address lines and 8 data lines to appear on the SMD SRAM memory is challenging. I will probably reassign the address lines to suit the Z8S180 pin-out, rather than spending hours untangling the lines. It won’t be too easy to do this with the Flash device, but as it is in a PLCC through-hole socket it comes inherently with “vias” making reaching it half as hard as the SMD device."

In the end I got the address and data lines to "flow" nicely around the board with very few vias, as this screenshot shows.

![address data](https://user-images.githubusercontent.com/3955592/36494015-0b60c444-1785-11e8-980e-bd3ce8acf50e.png)
